src/Xunit.NetCore.Extensions/Utilities.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Xunit.NetCore.Extensions
+{
+    internal static class Utilities
+    {
+#if NETFRAMEWORK
+        public static bool IsRunningOnNet => false;
+
+        public static bool IsRunningOnNetStandard => false;
+
+        public static bool IsRunningOnNetFramework => true;
+#elif NETSTANDARD
+        public static bool IsRunningOnNet => false;
+
+        public static bool IsRunningOnNetFramework => false;
+
+        public static bool IsRunningOnNetStandard => true;
```

Comment:
nit: .NET Standard is not really something to be running on. How about replacing `IsRunningOn...` with `IsBuiltAgainst...`?

Reply:
I removed `standard` - indeed it's never used. If needed - it can be added; and changed the property names as suggested.